src/server/routes/users.ts

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const controllers = require('../controllers.ts');
+const users = express.Router();
+
+
+users
+  .route('/:user_id')
+  .get(controllers.getOneUser)
```

Comment:
will also need a post request for new users after signing up

Reply:
very weird, i have a post route but it's not showing up here. will make sure it adds on on next commit